spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/RedisProperties.java

```diff
@@ -188,6 +191,14 @@ public Lettuce getLettuce() {
 		 */
 		private Duration maxWait = Duration.ofMillis(-1);
 
+		/**
+		 * Time of milliseconds to sleep between runs of the idle object evictor thread.
```

Comment:
As with the existing description for `maxWait`, the description shouldn't mention the unit (milliseconds in this case). Using `Duration` means that you can specify the time in any unit you want.